src/main/kotlin/dev/arbjerg/lavalink/client/loadbalancing/regionFiltering.kt

```diff
@@ -55,40 +72,46 @@ object RegionGroup {
         "ASIA" -> ASIA
         "EUROPE" -> EUROPE
         "US" -> US
+        "SOUTH_AMERICA" -> SOUTH_AMERICA
+        "AFRICA" -> AFRICA
         else -> throw IllegalArgumentException("No region constant: $region")
     }
 }
 
 // TODO In case no exact server match, should it look for the closest node in that same region?
-enum class VoiceRegion(val id: String, val visibleName: String) {
-    BRAZIL("brazil", "Brazil"),
-    HONGKONG("hongkong", "Hong Kong"),
-    INDIA("india", "India"),
-    JAPAN("japan", "Japan"),
-    ROTTERDAM("rotterdam", "Rotterdam"),
-    RUSSIA("russia", "Russia"),
-    SINGAPORE("singapore", "Singapore"),
-    SOUTH_AFRICA("southafrica", "South Africa"),
-    SIDNEY("sidney", "Sidney"),
-    US_CENTRAL("us-central", "US Central"),
-    US_EAST("us-east", "US East"),
-    US_SOUTH("us-south", "US South"),
-    US_WEST("us-west", "US West"),
-
-    UNKNOWN("", "Unknown");
+enum class VoiceRegion(val id: String, val subregions: HashSet<String>, val visibleName: String) {
```

Comment:
Instead of this hashset, I would prefer to just have separate region entries for all these other ones. 
Especially because Stockholm (in Sweden) is not even close to Rotterdam (in The Netherlands)

Reply:
I agree, I didn't know if you wanted to keep this set consistent with the regions in the UI, but I also prefer new regions, since for regions like Africa and middle east are also very far away